### PR TITLE
Use siginfo for easy Ctrl + T access on mac

### PR DESCRIPTION
**Pull request by emilbayes**
This makes it way more convenient on mac :)

### CODE REVIEW

File: include.js [hunk 1]
```diff
@@ -1,3 +1,3 @@
 var why = require('./')
 
-process.on('SIGUSR1', function() { why() })
```
Comment (mafintosh) [inline include.js:3]: did you mean to delete this line?

Comment (emilbayes) [reply include.js:3]: Yep, that's what `siginfo` does. Attaches a `SIGUSR1` and `SIGINFO` listener

Comment (mafintosh) [reply include.js:3]: i see the module does this, neat

File: include.js [hunk 2]
```diff
@@ -1,3 +1,3 @@
...
+require('siginfo')(why)
```
Comment (mafintosh) [inline include.js:3]: you should pass the force flag here for semver minor update

Comment (emilbayes) [reply include.js]: done

**Comment by mafintosh** [comment]
2.2.0

**Pull request merged**